tests/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -276,4 +276,17 @@ public function testQuotedTableNameRemainsQuotedInSchema(): void
         $columns = $schemaManager->listTableColumns('"tester"');
         self::assertCount(1, $columns);
     }
+
+    public function testReservedTableNameCommentRetrieval(): void
+    {
+        $this->connection->executeStatement('CREATE TABLE "users" ("users" NUMBER)');
+        $this->connection->executeStatement('COMMENT ON TABLE "users" IS \'table comment\'');
+        $this->connection->executeStatement('COMMENT ON COLUMN "users"."users" IS \'column comment\'');
```

Comment:
Should I clean up those `executeStatement`s on teardown? Or maybe I should scope them behind a `testschema`?

Reply:
Not applicable anymore